src/commands/experiments-clear.ts

```diff
@@ -0,0 +1,29 @@
+import { bold } from "colorette";
+
+import { Command } from "../command";
+import { FirebaseError } from "../error";
+import * as experiments from "../experiments";
+import { logger } from "../logger";
+import { last } from "../utils";
+
+export const command = new Command("experiments:clear <experiment>")
+  .description("clear preferences for an experiment on this machine")
+  .action((experiment: string) => {
+    if (experiments.isValidExperiment(experiment)) {
+      experiments.setEnabled(experiment, null);
+      experiments.flushToDisk();
+      logger.info(`Cleared preferences for experiment ${bold(experiment)}`);
+      return;
```

Comment:
Should this throw instead?

Reply:
Actually, Michael asked us to remove this feature for now.